<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_window</class>
 <widget class="QMainWindow" name="start_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Начальный экран</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 212, 121, 128);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_entre">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>60</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 212, 121, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ВХОД</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_login">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_logo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 212, 121, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Not programming code/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_login">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 212, 121, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 212, 121, 0);</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_entre">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 147, 0, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_registration">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 147, 0, 133);</string>
    </property>
    <property name="text">
     <string>Зарегистироваться</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Нет аккаунта?</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
